frontend-project/src/pages/tags/new/index.tsx

```diff
@@ -3,9 +3,10 @@ import { Button, Col, Card, Form, Input, Row } from 'antd';
 import { connect } from 'dva';
 import React, { useEffect, FC } from 'react';
 import { formatMessage, FormattedMessage } from 'umi-plugin-react/locale';
+import { Tag } from '@/services/definitions';
 
 interface TagNewFormProps {
-  name: String;
+  name: string;
   dispatch: Function;
```

Comment:
Function to bardzo ogólny interfejs, pasowałoby coś bardziej konkretnego

Reply:
Co proponujesz?

Reply:
Redux (i react-redux) ma to już otypowane. Wydaje mi się, że można po prostu dać `dispatch: Dispatch<AnyAction>`.

Reply:
@kuskoman  a w zasadzie jak nazwać ten nowy model ? 
Notification ?

Reply:
<img width="1436" alt="Zrzut ekranu 2020-10-25 o 09 57 56" src="https://user-images.githubusercontent.com/9075439/97102777-967ca500-16a8-11eb-9627-9abbe3237339.png">
Po przeniesieni metody do osobnego pliku rzuca takim błędem.

Reply:
@daxter44 spushuj to zobaczę

Reply:
@kuskoman https://github.com/watchdogpolska/small_eod/pull/658/commits/519fe3df20705abe5800c01f5281c7c22374f3c5 ten commit